radio/src/targets/horus/CMakeLists.txt

```diff
@@ -25,6 +25,9 @@ set(FLAVOUR horus)
 set(RAMBACKUP YES)
 set(LUA YES)
 set(PPM_LIMITS_SYMETRICAL YES)
+# for size report script
+set(CPU_TYPE_FULL STM32F429xI)
```

Comment:
I think the next version will parse a .ld script MEMORY section and determine the segments automatically... but I can't promise when that will be done.